ragmac_xdem/dem_postprocessing.py

```diff
@@ -332,9 +345,19 @@ def postprocessing_single(
     :returns: a tuple containing - basename of coregistered DEM, [count of obs, median and NMAD over stable terrain, coverage over roi] before coreg, [same stats] after coreg
     """
     # Load DEM and reproject to ref grid
+    
+    ref_dem = xdem.DEM(ref_dem)
     dem = xdem.DEM(dem_path)
+    common_bounds = gu.projtools.merge_bounds([ref_dem.bounds,dem.bounds],merging_algorithm='intersection')
+    common_bounds = {'left':common_bounds[0],
+                    'bottom':common_bounds[1],
+                    'right':common_bounds[2],
+                    'top':common_bounds[3]}
+    ref_dem = ref_dem.reproject(dst_res=ref_dem.res,dst_bounds=common_bounds,resampling='bilinear')
```

Comment:
The issue with that approach is that each DEM will have a different grid. DOes it not fail when stacking the results?
I thought your plan was to calculate the union of all ASTER DEMs, then take the intersection with the reference DEM.